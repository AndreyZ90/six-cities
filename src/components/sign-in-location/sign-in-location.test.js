import React from 'react';
import TestRenderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';

import SignInLocation from '@/components/sign-in-location/sign-in-location';

const city = `Amsterdam`;

describe(`SignInLocation component snapshot`, () => {
  test(`Should correctly render SignInLocation component`, () => {
    const tree = TestRenderer.create(
        <MemoryRouter>
          <SignInLocation city={city} />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
