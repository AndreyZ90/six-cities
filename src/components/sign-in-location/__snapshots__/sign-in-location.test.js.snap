// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInLocation component snapshot Should correctly render SignInLocation component 1`] = `
<section
  className="locations locations--login locations--current"
>
  <div
    className="locations__item"
  >
    <a
      className="locations__item-link"
      href="/amsterdam"
      onClick={[Function]}
    >
      <span>
        Amsterdam
      </span>
    </a>
  </div>
</section>
`;
